tests/unit/store/settings.spec.js

```diff
@@ -71,30 +71,30 @@ describe("store/settings", () => {
     expect(store.state.settings.customer_details.uuid).toBeNull();
     expect(store.state.settings.customer_details.api_key).toBeNull();
     expect(store.state.settings.customer_details.nickname).toBeNull();
-    expect(store.state.settings.customer_details.user_ids.length).toEqual(0);
+    expect(store.state.settings.customer_details.user_ids).toHaveLength(0);
     store.commit("settings/set_customer_details", array_of_customerids);
-    expect(store.state.settings.customer_details[0].cust_id).toEqual(0);
-    expect(store.state.settings.customer_details[0].uuid).toEqual(
+    expect(store.state.settings.customer_details[0].cust_id).toStrictEqual(0);
+    expect(store.state.settings.customer_details[0].uuid).toStrictEqual(
       "4vqyd62oARXqj9nRUNhtLQ"
     );
-    expect(store.state.settings.customer_details[0].api_key).toEqual(
+    expect(store.state.settings.customer_details[0].api_key).toStrictEqual(
       "941532a0-6be1-443a-a9d5-d57bdf180a52"
     );
-    expect(store.state.settings.customer_details[0].nickname).toEqual(
+    expect(store.state.settings.customer_details[0].nickname).toStrictEqual(
       "Customer account -1"
     );
-    expect(store.state.settings.customer_details[0].user_ids.length).toEqual(1);
+    expect(store.state.settings.customer_details[0].user_ids).toHaveLength(1);
     expect(
       store.state.settings.customer_details[0].user_ids[0].user_id
-    ).toEqual(0);
-    expect(store.state.settings.customer_details[0].user_ids[0].uuid).toEqual(
-      "2VSckkBYr2An3dqHEyfRRE"
-    );
+    ).toStrictEqual(0);
+    expect(
+      store.state.settings.customer_details[0].user_ids[0].uuid
+    ).toStrictEqual("2VSckkBYr2An3dqHEyfRRE");
     expect(
       store.state.settings.customer_details[0].user_ids[0].nickname
-    ).toEqual("User account -1");
+    ).toStrictEqual("User account -1");
   });
-  test("Store a single customer detail can be assigned to the customer_ids array", () => {
+  test("When initialized the array of customer_account_ids is empty and size 0, Then commit a single valid record of customer_details as the first record in customer_account_ids, customer_index/user_index and assert the same", () => {
```

Comment:
similar comment as previous test

Reply:
Would like to retain the present title, its more informative.